hindi/Akshay_Kumar/akshay_kumar.md

```diff
@@ -1,4 +1,4 @@
-Here are some facts given about Akshay Kumar, to know more 
-click [here](https://en.wikipedia.org/wiki/Akshay_Kumar)
-
+[Here](https://en.wikipedia.org/wiki/Akshay_Kumar) is a link to his wiki page.<br />
+[Here](https://www.facebook.com/akshaykumarofficial/) is a link to his facebook page.<br />
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.